package.json

```diff
@@ -128,6 +128,7 @@
     "license-checker": "^20.1.0",
     "minimatch": "^3.0.4",
     "minimist": "^1.2.0",
+    "nodemon": "^1.19.4",
```

Comment:
Unintentional?

Reply:
Removed the dependency and the reference in the documentation. I also removed the script in `package.json`.